### PR TITLE
zotero: update to 5.0.96.3

### PR DESCRIPTION
**Pull request by WhiredPlanck**
Topic Description
-----------------
Update `zotero` to 5.96.3

Package(s) Affected
-------------------

`zotero`

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
    

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`

### CODE REVIEW

**Comment by WhiredPlanck** [comment]
LGTM on AMD64

image

File: extra-productivity/zotero/autobuild/defines [hunk 1]
```diff
@@ -1,8 +1,5 @@
 PKGNAME=zotero
-PKGDES="A free, easy-to-use tool to help you collect, organize, cite, and share your research sources"
+PKGDES="A free tool to help you collect, organize, cite, and share your research sources"
 PKGSEC=doc
 PKGDEP="dbus-glib alsa-lib gtk-3 gcc-runtime nss"
 BUILDDEP="nodejs"
-
-FAIL_ARCH="!(amd64)"
```
Comment (cthbleachbit) [inline extra-productivity/zotero/autobuild/defines:7]: Note that you have removed `FAIL_ARCH` from `defines`, but left archteciture specific codes in `build`.

I don't think this works for anything other than `amd64` as zotero uses prebuilt `xulrunner` and `pdftools`. Either:
- Leave `FAIL_ARCH="!(amd64)"` in place, or
- Try to see if this works with in-house built pdftools and xulrunner (part of firefox).

**Review by cthbleachbit — CHANGES_REQUESTED** [review]

File: extra-productivity/zotero/autobuild/defines (hunk 1 above)
Comment (WhiredPlanck) [reply extra-productivity/zotero/autobuild/defines:7]: I see, I will try to figure out that. Thanks for your kindness ~.

**Review by cthbleachbit — APPROVED** [review]
LGTM

**Pull request merged**